spec.html

```diff
@@ -2465,34 +2455,6 @@ <h1>
           </emu-alg>
         </emu-clause>
 
-        <emu-clause id="sec-numeric-types-bigint-sameValue" type="numeric method">
-          <h1>
-            BigInt::sameValue (
-              _x_: a BigInt,
-              _y_: a BigInt,
-            ): a Boolean
-          </h1>
-          <dl class="header">
-          </dl>
-          <emu-alg>
-            1. Return BigInt::equal(_x_, _y_).
-          </emu-alg>
-        </emu-clause>
-
-        <emu-clause id="sec-numeric-types-bigint-sameValueZero" type="numeric method">
```

Comment:
these IDs both need to be oldIDs somewhere

Reply:
Good catch. Done.